src/Illuminate/Database/Query/Builder.php

```diff
@@ -1562,7 +1564,7 @@ public function exists()
 	 * Retrieve the "count" result of the query.
 	 *
 	 * @param  string  $columns
-	 * @return int
+	 * @return mixed
```

Comment:
@JoostK one more question
in min - it will be min of the grouped min's
in max - max of max's
in avg - avg of avg's
and in count - hmm... count of lines